app/components/Quote.tsx

```diff
@@ -7,9 +7,9 @@ interface QuoteProps {
   author: string;
 }
 
-export default function Quote({ text, author }) {
+export default function Quote({ text, author }: QuoteProps) {
   return (
-    <blockquote className="mb-10 p-12 border-l-8 border-purple-600 bg-purple-300">
+    <blockquote className="mb-10 p-12 bg-transparent">
```

Comment:
Remove unneeded class
```suggestion
    <blockquote className="mb-10 p-12">
```